editor/src/ui-tools/quick-action/quick-action-ui.ts

```diff
@@ -71,7 +73,7 @@ export class QuickActionUI extends AbstractUIExtension implements IActionHandler
   }
 
   selectionChanged(root: Readonly<SModelRoot>, selectedElements: string[]): void {
-    if (selectedElements.length < 1) {
+    if (this.editorContext.modelRoot.cssClasses?.includes(CursorCSS.MARQUEE) || selectedElements.length < 1) {
```

Comment:
Looks very hacky! No Action for this available?

Reply:
Only a combined EnableToolsAction with the marquee id. But it's tested so enough for the moment I think